tests/packages/test_locker.py

```diff
@@ -86,7 +86,7 @@ def test_lock_file_data_is_ordered(locker, root):
 [metadata]
 lock-version = "1.1"
 python-versions = "*"
-content-hash = "115cf985d932e9bf5f540555bbdd75decbb62cac81e399375fc19f6277f8c1d8"
+content-hash = "178f2cd01dc40e96be23a4a0ae1094816626346346618335e5ff4f0b2c0c5831"
```

Comment:
It would be interesting to create a constant for `content-hash`. Here we see an example that if it had a constant, it would not be necessary to make 7 changes to the same value.